sources/install/package_ad.sh

```diff
@@ -424,7 +437,9 @@ function install_polenum() {
 
 function install_smbmap() {
     colorecho "Installing smbmap"
-    python3 -m pipx install git+https://github.com/ShawnDEvans/smbmap
+    # python3 -m pipx install git+https://github.com/ShawnDEvans/smbmap
+    # https://github.com/ShawnDEvans/smbmap/pull/86
+    python3 -m pipx install git+https://github.com/reversebrain/smbmap
```

Comment:
Maybe instead of using a fork, we should merge dynamically the PR https://github.com/ShawnDEvans/smbmap/pull/86
Using a forked repo not trusted might be security risk

Reply:
merge dynamically the PR ?

Reply:
> Maybe instead of using a fork, we should merge dynamically the PR [ShawnDEvans/smbmap#86](https://github.com/ShawnDEvans/smbmap/pull/86) Using a forked repo not trusted might be security risk

I don't think creating a fork only for this change is relevant. We could hot-merge the PR during the build process, something like

```bash
git -C /opt/tools/ clone https://github.com/ShawnDEvans/smbmap
cd /opt/tools/smbmap/
git config --global user.email "exegol@install.er"
git config --global user.name "Exegol installer"
git fetch origin pull/86/head:pull/86
git merge --no-edit pull/86
python3 -m pipx install .
```

Reply:
This does not solve the security problem. if there are new commits they will be merged

Reply:
if you don't want to need to trust a fork you can modify the dependencies to hot as for `pywsus`

Reply:
Indeed. Other option is to download the .patch, store it in `/sources/assets/patches/` and apply it during the build process (`git apply --verbose <file.patch>`).

Reply:
[27c6aa5](https://github.com/ThePorgs/Exegol-images/pull/216/commits/27c6aa5f7ca62717980def6e5202f65beb427ca6)